test/IISIntegration.IISServerFunctionalTests/IISIntegration.IISServerFunctionalTests.csproj

```diff
@@ -1,8 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
```

Comment:
Revert. Many of the NTLM tests require net461 clients.

Reply:
This is a separate set of tests I created. ANCM in proc can't run on net461 right?

Reply:
The test runs as net461, but it always starts IIS Express for a Core app. The two are unrelated.